etc/config/hlsl.defaults.properties

```diff
@@ -0,0 +1,3 @@
+compilers=/usr/dxc-artifacts/bin/dxc
```

Comment:
This code will only work for default users. Once we have a corresponding infra PR, we can see what we need to add into an eventual `hlsl.amazon.properties` file

Reply:
We can do this change once things are built ok!

Reply:
OK for now I've removed this file and we can add things back once binaries are available through s3